js/background.js

```diff
@@ -38,11 +19,15 @@ chrome.browserAction.onClicked.addListener(function (tab)
 			url: chrome.extension.getURL("TabsAside.html"),
 			active: true
 		},
```

Comment:
Sorry, I just don't get it. Why it should be inserted into a lambda function?

Reply:
Don't apologize, it could mean I'm in the wrong on this one.

I basically believe we should do 
```
functionWithCallback(args, Callback)
```
```
chrome.tabs.create({...}, Callback)
```
and not
```
functionWithCallback(args, Callback(callbackArgs))
```
```
chrome.tabs.create({...},Callback(...))
```

Without the lambda, we're doing exactly this, with Callback(...) being the whole `chrome.tabs.onActivated.addListener(...)` call on line 22. 

Tell me if it's still not clear, and I'll come back and try to explain better after a good night sleep

Reply:
Ok. I've got what you want to do. But I still see no difference between these two. In fact, the first one block of code seems cleaner to me

Reply:
I agree that the first one seems cleaner - but according to the language, it would be incorrect, as a callback should be a **function object** `Callback`, and not the **value** `Callback(...)` (which gets the results of the callback call, and sends that result to the function).

Or at least that's how I think callback works. I'll have to check, as I said I may be confusing languages, but I feel like this is it

Reply:
Various documentations seems to confirm my claims (small [MDN](https://developer.mozilla.org/en-US/docs/Glossary/Callback_function), long [MDN](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Introducing#Async_callbacks) ) :

We have to pass a function object as argument. In our case, instead of passing the callback function, we call it ourselves, get its results (= `void`) and pass `void` as the callback function.

The main side effect of that mistake is that in our case the callback is called **before** the chrome.tabs.create(...) call, instead of *after*.

Reply:
Ok then. I'll add this